include/CL/opencl.hpp

```diff
@@ -9137,6 +9231,66 @@ typedef CL_API_ENTRY cl_int (CL_API_CALL *PFN_clEnqueueReleaseD3D10ObjectsKHR)(
         return detail::errHandler(::clFinish(object_), __FINISH_ERR);
     }
 
+#if defined(cl_khr_external_memory) && CL_HPP_TARGET_OPENCL_VERSION >= 300
+    cl_int EnqueueAcquireExternalMemObjects(
+        const vector<Memory>& mem_objects = {},
+        const vector<Event>& events_wait = {},
```

Comment:
I think this is better, but I also think it's important to be consistent, so we should probably use

```c++
const vector<Event>* events = nullptr
```
here instead.  Same goes for `enqueueReleaseExternalMemObjects`.

Reply:
Agree, I've applied the change. Thanks!